### ISSUE TITLE
RegEx not matching

### ISSUE THREAD

**Issue by AnnaKalkanis** [jameslnewell/xhr-mock#20]
When using RegEx, my request is not matching mock. 
- Request against 'https://localhost.fake/api/uploads/test/123456'. 

- For mock, I've tried 

```
 mock.mock('PUT', /.*\/uploads\/.*/,
      (req, res) => res.status(200).body("")
);
```

and 

```
mock.put(/.*\/uploads\/.*/,
      (req, res) => res.status(200).body("")
);
```

It works fine when I use the actual url, but does not seem to match against various valid RegExs that I've tried. Am I formatting this incorrectly? Is it possible to use a wildcard?

**Comment by AnnaKalkanis** [comment]
Looks like it's just a versioning issue. Any idea when the latest code with Regex matching will be available on npm? I see it in the source here, but it's not in the version I've installed from npm.

**Comment by jameslnewell** [comment]
Sorry about that! I think I was waiting on https://github.com/jameslnewell/xhr-mock/pull/16 to be complete which adds similar functionality and some bug fixes. I've published `1.8.0` in the mean time.

Let me know if you have any more issues!
